cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml

```diff
@@ -147,7 +147,7 @@ components:
                         type: curie
                         to_column: assay
             disease_ontology_term_id:
-                error_message_suffix: "Only 'PATO:0000461' is allowed for 'PATO' term ids."
+                error_message_suffix: "Only 'PATO:0000461' (normal), children terms of 'MONDO:0000001' (disease), and self or children terms of 'MONDO:0021178' (injury) are allowed"
```

Comment:
> children terms

children shouldn't be used an adjective; should be "child terms". I'm noticing we have this phrasing issue across the schema definition. I'll defer to @brianraymor on whether or he likes this phrasing as-is, doesn't care, or wishes to see it changed throughout.

Reply:
Also this is a single term, so "and" should become "or"

Reply:
```suggestion
                error_message_suffix: "Only 'PATO:0000461' (normal), 'MONDO:0021178' (injury) or child terms thereof, or child terms of 'MONDO:0000001' (disease) are allowed"
```